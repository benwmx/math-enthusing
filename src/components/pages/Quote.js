import React from 'react';

const Quote = () => (
  <div className="container">
    <h2 className="title">Quote of the Day</h2>
    <p className="quote">
      &quot;Mathematics is not about numbers, equations, computations,
      or algorithms: it is about understanding.&quot;
      ~William Paul Thurson.
    </p>
  </div>

);

export default Quote;
