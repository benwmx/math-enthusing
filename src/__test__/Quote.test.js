import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../components/pages/Quote';

describe('Quote component test', () => {
  it('Should render', () => {
    const quote = renderer.create(<Quote />).toJSON();
    expect(quote).toMatchSnapshot();
  });
});
