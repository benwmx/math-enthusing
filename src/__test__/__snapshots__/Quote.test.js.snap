// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component test Should render 1`] = `
<div
  className="container"
>
  <h2
    className="title"
  >
    Quote of the Day
  </h2>
  <p
    className="quote"
  >
    "Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding." ~William Paul Thurson.
  </p>
</div>
`;
